### DISCUSSION TITLE
Does Tempest include any API for creating directories in an OS-independent manner?

### DISCUSSION THREAD

**Discussion by Newbytee** [Try/OpenGothic#333] [Q&A] [answered]
For the feature I'm working on for OpenGothic (saving savegames outside of the current working directory) I need to create leading directories to ensure that the files actually can be created where specified. In the answer to my previous question (https://github.com/Try/OpenGothic/discussions/331#discussioncomment-3778496), `Tempest::RFile` and `Tempest::WFile` were mentioned, but as far as I can tell these are for working on files and not directories. Does Tempest include any infrastructure for creating directories in an OS-independent manner?

**Comment by Try** [comment, marked as answer]
Hi, @Newbytee !

No, it doesn't. `std::filesystem` should be fine, since we are moving to C++20
